### DISCUSSION TITLE
Could this work in a browser?

### DISCUSSION THREAD

**Discussion by GP4cK** [ethanblake4/dart_eval#50] [Q&A] [answered]
What would it take to make it work in the browser?

**Comment by ethanblake4** [comment, marked as answer]
v0.5 is coming out soon and will work fully in the browser. There was a Dart SDK issue preventing it, https://github.com/dart-lang/sdk/issues/50048, which the Dart team has fixed :)

**Reply by ethanblake4** [reply]
@GP4cK v0.5 is out now with Web support
